data-non-standard.js

```diff
@@ -1218,7 +1212,7 @@ exports.tests = [
     firefox12: false,
     firefox16: true,
     firefox17: false,
-    firefox20: true,
+    firefox20: false,
     firefox28: false,
```

Comment:
> Apparently having these 2 redundant lines mean that both browsers were actually tested.

> I kept both because I didn't wat to have my patch refused again because of the optimization.

All browsers which have identical `full` field will nest data from previous marked version's value for each test and it's normal in this repo. Actually you loose data when you delete browser from browser list and not loose any useful data when you delete repeating values in tables (and especially if you delete data for browser not represented in browser list).

> These browsers ARE rendered in this table if you check the "Obsolete Browsers" checkbox. For the other tables, the data is there and is wrong, so I believe it is worth fixing even if they are not displayed by the default renderer.

There are no Firefox browsers with versions lower then 31 get rendered if you check obsolete browsers checkbox on. This is because of `obsolete: very,` fields in very old versions. There is nothing about very obsolete browsers in DOM. So this is not really an optimisation.